src/test/java/org/jitsi/meet/test/util/MeetUtils.java

```diff
@@ -36,7 +36,7 @@
      * is in state 'connected'.
      */
     public static final String ICE_CONNECTED_CHECK_SCRIPT =
-        "return APP.conference.getConnectionState() === 'connected';";
+        "return APP.conference.getP2PConnectionState() === 'connected';";
```

Comment:
no, this script is supposed  to check if conference's ICE state is connected and not just  P2P